### DISCUSSION TITLE
Feat: group the checks in the PR by each yaml file

### DISCUSSION THREAD

**Discussion by jeffydc** [community/community#26425] [Actions]
<p>Hi,<br>
First and foremost, I am not sure I am requesting feature in the appropriate place.</p>
<p>For minimal example, we have 3 workflows files, 1 of them is python unit testing with matrix of 3 different OSes with 3 different python versions, 1 of them is docker building with 3 different python versions, the rest is python package building testing in 3 different OSes with 3 different python versions. So, the total checks are 21, which is kinda hard to see the failed job in a glance in the PR checks. I could combine some of them into a single workflow, but they serve a specific part of testing.</p>
<p>So, I would like to request if GitHub can provide grouping the jobs in the PR checks by each workflow yaml file?</p>
<p>Thank you!</p>

**Comment by BrightRan** [comment, marked as answer]
<p><a class="mention" href="/ydcjeff">@ydcjeff</a>,</p>
<p>I recommend you directly share a feature request in the <a href="https://support.github.com/contact/feedback" rel="noopener nofollow ugc"><strong>Feedback form for GitHub</strong></a>.<br>
That will allow you to directly interact with the appropriate engineering team, and make it more convenient for the engineering team to collect and categorize your suggestions.</p>

**Comment by jeffydc** [comment]
<p>Thank you <a class="mention" href="/brightran">@brightran</a></p>
